app/src/components/main/paginationToolbar/paginationToolbar.jsx

```diff
@@ -1,5 +1,15 @@
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
+import {
+  LEVEL_LAUNCH,
+  LEVEL_SUITE,
+  LEVEL_TEST,
+  LEVEL_STEP,
+  LEVEL_LOG,
+  LEVEL_NOT_FOUND,
+} from 'common/constants/launchLevels';
+import { getPageSize, setPageSize } from 'controllers/log';
```

Comment:
Paging is under responsibility of `controllers/pagination`.

Reply:
Fixed.